plugins/meta/tuning/tuning.go

```diff
@@ -542,3 +545,25 @@ func readAllowlist() (bool, []string, error) {
 	}
 	return true, allowList, nil
 }
+
+type sysctlKey string
+
+type sysctlCheck struct {
+	SysCtl map[sysctlKey]string `json:"sysctl"`
+}
+
+var sysctlDuplicatesMap = map[sysctlKey]interface{}{}
+
+func (d *sysctlKey) UnmarshalText(data []byte) error {
+	key := sysctlKey(string(data))
+	if _, exists := sysctlDuplicatesMap[key]; exists {
+		return errors.New("duplicated sysctl keys are not allowed")
+	}
+	sysctlDuplicatesMap[key] = ""
+	return nil
+}
+
+func validateSysctlConflictingValues(data []byte) error {
```

Comment:
Actually this function is checking conflicting keys not values, i think it should be renamed?

Reply:
indeed, fixed